### PR TITLE
NR-294934 alert deprecation

### PR DESCRIPTION
**Pull request by mickeyryan42**
https://new-relic.atlassian.net/browse/NR-294934

### Summary

Removing collapser that explains how to install standalone alerts, these are being deprecated.

### CODE REVIEW

File: src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import journeyAlertsAndDash from 'images/dashboards_screenshot-crop_journey-dash
...
+Where New Relic entities can give you scores of data to observe, our dashboards and <InlinePopover type="alerts" /> come in to help you make the most of it. You can use our pre-configured options to set up dashboard solutions out of the box, or use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), New Relic's Query Language, to create custom dashboard components and alert policies that are as fine-tuned to your requirements as you need want them to be. And like all our other features, you can try out dashboards and alerts using your own data [for free](https://newrelic.com/signup).
```
Comment (josephgregoryii) [inline src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx:25]:
```md
<InlinePoperover type="alerts" />
```
unsure if this needs to be updated or not. is the intent of this to add alerts, or just dashboards, or both via quickstarts?

File: src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx [hunk 2]
```diff
@@ -96,54 +96,14 @@ Our dashboards take the data you put into New Relic and provide you with a wide
 
 ## Your alerting solution [#alerts-solutions]
 
-Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize alerts for your unique requirements while helping to eliminate alert fatigue. We offer proactive awareness for your problems, utilizing machine learning to enhance our policies. Our alerts:
+Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize <InlinePopover type="alerts" /> for your unique requirements while helping to eliminate alert fatigue. We offer proactive awareness for your problems, utilizing machine learning to enhance our policies. Our alerts:
```
Comment (josephgregoryii) [inline src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx:99]: I think same here?

**Review by josephgregoryii — COMMENTED** [review]
Couple questions, but not necessarily blocking. Just want to double check the intent for the docs to make sure we're supplying the right information :)

File: src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx:25]: That popover code just points people to the alerts UI so they don't have to hunt it down themselves.

File: src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx:99]: It's just providing an easy link into the UI. We throw these in across the docs site. It's just a convenience for people who may not know where things are in the UI. This seems useful particularly as we're removing alerts from the install flow in the UI.

**Review by sarahkitten — COMMENTED** [review]
I think there's a couple more pages that should be updated!
https://docs.newrelic.com/docs/tutorial-dd-migration/installing-monitor/
https://docs.newrelic.com/install/kubernetes/

**Comment by mickeyryan42** [comment]
> https://docs.newrelic.com/docs/tutorial-dd-migration/installing-monitor/

Great catch @sarahkitten!  @paperclypse would you mind taking a look at these two pages?  They reference the alerts tab so i think we definitely want to update/delete them.  If you think we can just remove them i can go ahead and do so, but please let me know if theres anything specific i should update these with.

File: src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx (hunk 2 above)
Comment (josephgregoryii) [reply src/content/docs/tutorial-dd-migration/migrating-dashboards.mdx:99]: Thanks for the clarification! :)

**Review by paperclypse — APPROVED** [review]
Looks good!

**Comment by paperclypse** [comment]
@mickeyryan42 we had a big global image update across our docs site. I had to deal with some merge conflicts.

**Review by paperclypse — APPROVED** [review]
OK, looks good now!

**Pull request merged**